src/main/java/org/braekpo1nt/mctmanager/games/mecha/MechaGame.java

```diff
@@ -335,7 +343,23 @@ public void onPlayerDeath(PlayerDeathEvent event) {
             onTeamWin(winningTeam);
         }
     }
-    
+
+    @EventHandler // prevent player interaction with Item Frames or Armor Stands
+    public void onPlayerInteract(PlayerInteractEvent event) {
+        if (!gameActive) {
+            return;
+        }
+        if (!mechaHasStarted) {
+            return;
+        }
```

Comment:
You need to do a check for if the player associated with the event is in the list of participants

Reply:
Check!